config/examples/settings.yml

```diff
@@ -36,17 +32,24 @@ base: &default
   onpremises_api_docs_version: false
   assets_cdn_host: <%= ENV.fetch('ASSETS_CDN_HOST', '') %>
   email_sanitizer:
-    enabled: <%= ENV.fetch('EMAIL_SANITIZER_ENABLED', Rails.env.preview?) %>
-    to: <%= ENV.fetch('EMAIL_SANITIZER_TO', 'saniziter@example.com') %>
+    enabled: <%= ENV.fetch('EMAIL_SANITIZER_ENABLED', false) %>
```

Comment:
Also this seems not well controllable with ENV variable. I htink it makes sense to disable when env variable is empty.

Reply:
Right. I'll use the same approach as discussed [here](https://github.com/3scale/porta/pull/3225/files#r1123097194) for now, and we can change it later.